pkg/apis/v1alpha1/register.go

```diff
@@ -75,6 +75,9 @@ var (
 	LabelInstanceGPUCount                     = LabelDomain + "/instance-gpu-count"
 	LabelInstanceGPUMemory                    = LabelDomain + "/instance-gpu-memory"
 	LabelInstanceAMIID                        = LabelDomain + "/instance-ami-id"
+	LabelInstanceAcceleratorName              = LabelDomain + "/instance-accelerator-name"
```

Comment:
Do we still need a concept of `gpu` now that we have the concept of `accelerator`?

Reply:
Is `accelerator-memory` something we should expose?

Reply:
One option is to continue to support gpu labels, but remove them from the docs.

Reply:
> Do we still need a concept of `gpu` now that we have the concept of `accelerator`?

Leaving the gpu label until v1beta1 for depracation